### DISCUSSION TITLE
how do i add folders

### DISCUSSION THREAD

**Discussion by LISPlays** [community/community#43786] [New to GitHub]
when i click the folder in my files it uploads as a empty file

**Comment by radar-blip** [comment]
Github doesn't support empty folders, there must be something in the folder for the folder to be created.

See the answer [here](https://stackoverflow.com/questions/12258399/how-do-i-create-a-folder-in-a-github-repository) for better understanding.

**Reply by LISPlays** [reply]
its not an empty folder
